src/app/GitUI/Editor/Diff/AnsiEscapeUtilities.cs

```diff
@@ -27,24 +38,14 @@ public static void PrintColors(StringBuilder sb, List<TextMarker> textMarkers)
         // color id (standard) colors
         foreach (bool fore in new List<bool>() { true, false })
         {
-            foreach (int offset in new List<int>() { 0, _boldOffset })
+            foreach (bool bold in new List<bool>() { false, true })
```

Comment:
`List<bool>() { false, true }` should be a static instance to avoid constant allocations.

Reply:
I changed this as code example.
But this code snippet will maybe be used by 10 persons at very few occasions to debug colors and tune the themes. So the static allocation will slow down a lot more than it helps. (I would prefer a comment.)